src/Middleware/RequestThrottling/RequestThrottling.slnf

```diff
@@ -16,10 +16,11 @@
       "..\\http\\Headers\\src\\Microsoft.Net.Http.Headers.csproj",
```

Comment:
This should be deleted now, right?

Reply:
Make sure that the RequestThrottling folder is completely removed.

Reply:
Need to rename and move the slnf file.

Reply:
yes it should